assets/js/feature-tours/pagespeed-insights-force-active.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Page Speed Insight Force Active tour for the dashboard.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import {
+	VIEW_CONTEXT_DASHBOARD,
+	VIEW_CONTEXT_PAGE_DASHBOARD,
+} from '../googlesitekit/constants';
+import {
+	MODULES_PAGESPEED_INSIGHTS,
+	STRATEGY_DESKTOP,
+	STRATEGY_MOBILE,
+} from '../modules/pagespeed-insights/datastore/constants';
+import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
+
+const pagespeedInsightsForceActive = {
+	slug: 'pagespeedInsightsForceActive',
+	contexts: [ VIEW_CONTEXT_DASHBOARD, VIEW_CONTEXT_PAGE_DASHBOARD ],
+	version: '1.46.0',
+	gaEventCategory: ( viewContext ) =>
+		`${ viewContext }_pagespeed-widget-force-active`,
+	checkRequirements: async ( registry ) => {
+		// Wait until page speed insights module has loaded.
+		const referenceURL = registry
+			.select( CORE_SITE )
+			.getCurrentReferenceURL();
+		await registry
+			.__experimentalResolveSelect( MODULES_PAGESPEED_INSIGHTS )
+			.getReport( referenceURL, STRATEGY_MOBILE );
+		await registry
+			.__experimentalResolveSelect( MODULES_PAGESPEED_INSIGHTS )
+			.getReport( referenceURL, STRATEGY_DESKTOP );
```

Comment:
Is this really needed? We point to the overall widget area, so I don't think it matters for the tooltip where the widget is fully loaded. I'd say let's remove this, less code to maintain.

Reply:
See my comment in https://github.com/google/site-kit-wp/issues/4133#issuecomment-965912725 for the issues with each approach.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/6e07da780fa8d2be9e85a194dc55592b879d4b07 addresses the CR comment.